src/lib/commands/remoteFs.ts

```diff
@@ -26,6 +30,7 @@ export const registerCommands = (context: ExtensionContext) => {
     commands.registerCommand(
       COMMANDS.REMOTE_FS.FETCH,
       async (clickedFileLink) => {
+        showMissingAccountError();
```

Comment:
If no account is specified, we throw an error to the user and exit the command. This guarantees that `getAccountId()` is not undefined.